Sources/Navidux/Core/Navigation/NavigationScreen/NavigationScreen.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 
 /// Main element of Navigation on Redux (Navidux). Uses for store/move screens in navigation stack.
-public protocol NavigationScreen: UIViewController, AnyObject, NavigationRestructable where Self: Equatable {
```

Comment:
Give me advise, why we remove equitable from navigation screen?

Reply:
Typo, sorry